### PR TITLE
ADAPT-4069: Tweak campaign hero logo max width and padding

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Fix max width on campaign logo on the fullwidth campaign page variant

# Review By (Date)
- 11/17 if possible since Patrick would like this in production by 11/20

# Criticality
- 9/10
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Pull down locally, nvm use, npm run develop
2. Go to http://app.storyblok.com/#!/me/spaces/78141/stories/0/0/77696947
2. Preview the page on mobile breakpoint, and check that the campaign logo doesn't overflow the box in the hero

Screen Shot 2021-11-16 at 4 26 23 PM

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/scss/page-types/_campaign-page--fullwidth-image.scss [hunk 1]
```diff
@@ -21,12 +21,16 @@
     }
 
     .campaign-page__hero-content-logo {
-      max-width: 43rem;
+      max-width: 100%;
+
+      @include grid-media('2xl') {
+        max-width: 43rem;
+      }
     }
```
Comment (yvonnetangsu) [inline src/scss/page-types/_campaign-page--fullwidth-image.scss:29]: max-width: 43rem only makes sense for 2xl breakpoint

File: src/scss/page-types/_campaign-page--fullwidth-image.scss [hunk 2]
```diff
@@ -21,12 +21,16 @@
...
 
     .campaign-page__hero-content {
       &-wrapper {
-        @include padding(3.4rem 2rem);
+        @include padding(3.4rem);
```
Comment (yvonnetangsu) [inline src/scss/page-types/_campaign-page--fullwidth-image.scss:33]: Padding for the campaign hero content box was handled strangely so cleaned that up a bit as well

**Review by sherakama — APPROVED** [review]
LG2M. GTG.

**Comment by yvonnetangsu** [comment]
> LG2M. GTG.

So speedy thank you @sherakama 😄

**Pull request merged**